README.md

```diff
@@ -0,0 +1,88 @@
+[![CocoaPods](https://img.shields.io/cocoapods/v/Iterable-iOS-SDK.svg?style=flat)](https://cocoapods.org/pods/Iterable-iOS-SDK)
+[![License](https://img.shields.io/cocoapods/l/Iterable-iOS-SDK.svg?style=flat)](https://opensource.org/licenses/MIT)
+[![Docs](https://img.shields.io/cocoapods/metrics/doc-percent/Iterable-iOS-SDK.svg?style=flat)](http://cocoadocs.org/docsets/Iterable-iOS-SDK)
```

Comment:
Cocoadocs are deprecated, so no docs will be generated for this pod. I guess we should remove this until we start hosting the generated docs somewhere.